Obsidian/Plugins/PluginProviders/RemotePluginProvider.cs

```diff
@@ -34,13 +29,13 @@ public async Task<PluginContainer> GetPluginAsync(string path, ILogger logger)
 
         return url.Host switch
         {
-            "www.github.com" => await LoadGithubPluginAsync(path, logger, repository),
-            "gist.github.com" => await LoadGistPluginAsync(path, logger, repository),
+            "www.github.com" => await LoadGithubPluginAsync(path, repository),
+            "gist.github.com" => await LoadGistPluginAsync(path, repository),
             _ => Failed("'unknown'", path, logger, $"Remote source {url.Host} is not supported")
```

Comment:
Might be cool to at some point allow "unsafe hosts" via a config variable

```json
{
    "ALLOW_UNSAFE_HOSTS_FOR_COMPILED_PLUGINS_LOOKOUT_DANGER_YIKES": true
}
```
idk